src/avatar.h

```diff
@@ -139,6 +140,10 @@ class avatar : public player
          */
         void on_mission_finished( mission &cur_mission );
 
+        // Dialogue and bartering--see npctalk.cpp
+        void talk_to( std::unique_ptr<talker> talk_with, bool text_only = false,
+                      bool radio_contact = false );
```

Comment:
Yeah, talk_to_u originally took two bools: text_only, so that certain aspects of it get ignored for testing purposes, and radio_contact, for conversations over a radio.  I might go back and fix them but not in this PR.